### DISCUSSION TITLE
How to run on Uno WinUI?

### DISCUSSION THREAD

**Discussion by FrozDark** [Mapsui/Mapsui#1684] [Q&A] [answered]
There is no nuget package. Even after I referenced to mapsui projects I couldn't handle to start any platform of Uno but WinUI

Could someone be pleasant to write some manual for that?

**Comment by inforithmics** [comment, marked as answer]
Uno WinUI is not officially released yet. 

Until it is released you can download it from the appveyor builds

This is the last build
https://ci.appveyor.com/project/pauldendulk/mapsui/builds/44335171/artifacts

**Comment by FrozDark** [comment]
For those who are out of knowledge I'll write a simple instruction installing from appveyor builds
1) Follow the link for the builds. 
2) Download nupkg files including the core MapsUI one and place them in one folder
3) From the Nuget Packacge Cli add new source of packages linking it to the folder you located the nupkg files
4) Don't forget to enable "Show preview builds" and install as ordinary nuget package
